cibuildwheel/util.py

```diff
@@ -1,5 +1,11 @@
 from fnmatch import fnmatch
 
+filter_wheels_cmd = '''\
+import sys, glob, itertools, pip; \
+sys.stdout.write(\
+' '.join(f for f in itertools.chain(*map(glob.iglob, sys.argv[1:])) \
+if pip.wheel.Wheel(f).supported()))\
+'''
```

Comment:
Sorry, I had missed that previous comment! Yes I think you're right in that respect.